### PR TITLE
Internal: Editor Preview - Allow to render a JSON beside current document [ED-13872]

### PR DESCRIPTION
**Pull request by matipojo**
Support the new AI Copilot to render suggestions beside current elements.

### CODE REVIEW

File: assets/dev/js/editor/editor-base.js [hunk 1]
```diff
@@ -506,7 +514,16 @@ export default class EditorBase extends Marionette.Application {
 	initPreviewView( document ) {
 		elementor.trigger( 'document:before:preview', document );
 
-		const preview = new Preview( { el: document.$element[ 0 ], model: elementor.elementsModel } );
+		this.previewView = this.renderPreviewFromJSONModel( document.$element[ 0 ],	elementor.elementsModel );
```
Comment (StyleShit) [inline assets/dev/js/editor/editor-base.js:517]: calling it "render" seems kinda weird when it also returns a value.
Not sure if I have a better name though... WDYT?

File: assets/dev/js/editor/views/preview.js [hunk 2]
```diff
@@ -3,12 +3,20 @@ import AddSectionView from './add-section/independent';
 const BaseSectionsContainerView = require( 'elementor-views/base-sections-container' );
 
 const Preview = BaseSectionsContainerView.extend( {
+	config: {
+		allowAddElements: true,
```
Comment (StyleShit) [inline assets/dev/js/editor/views/preview.js:7]:
```suggestion
		allowAddingElements: true,
```

maybe?

File: assets/dev/js/editor/editor-base.js (hunk 1 above)
Comment (matipojo) [reply assets/dev/js/editor/editor-base.js]: Done

File: assets/dev/js/editor/views/preview.js (hunk 2 above)
Comment (matipojo) [reply assets/dev/js/editor/views/preview.js]: Done

**Review by DennisNerush — APPROVED** [review]

**Pull request merged**